.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/python:3.7
+
+    working_directory: ~/cairo
+
+    steps:
+      - checkout
+      
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "requirements.txt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+      - run:
+          name: requirements
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -r requirements.txt
```

Comment:
why using an extra file when you could define this in one line?
pip install numpy matplotlib pandas pytest

Reply:
It's easier to update this file to make sure the requirements are met automatically. I want to minimize the amount I have to edit the ``config.yml`` file

Reply:
Sounds good.